model/SupplyChain/Classes/DefinedStateProcess.md

```diff
@@ -0,0 +1,26 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# DefinedStateProcess
+
+## Summary
+
+This process is used to determine the state of an affected Element.
+
+## Description
+
+This process is used to determine the state of an affected Element. 
+The DefinedStateProcess used to define a list of vaild state(s) of an affected Element.
+A DefinedStateProcess may describes the steps or conditions required to move an entity from one state to another. 
```

Comment:
```suggestion
The DefinedStateProcess is used to define a list of valid states of an affected Element.

A DefinedStateProcess may describe the steps or conditions required to move an entity from one state to another.
```

Fix minor spelling/grammar. Also suggest to split paragraphs to have one core message per paragraph.